src/applications/edu-benefits/tests/0994/utils.unit.spec.js

```diff
@@ -21,6 +21,15 @@ describe('0994 utils', () => {
 
       expect(maskedBankInfo.text()).to.contain('●●●●●ending with0000');
 
+      maskedBankInfo.unmount();
+    });
+  });
+  describe('maskBankInformation string is blank', () => {
+    it('should mask all but last 4 characters', () => {
```

Comment:
Should it mask all but the last 4? Or should it ensure that the text contains `is blank`?